docs/development/setup-vagrant.md

```diff
@@ -1062,6 +1062,10 @@ HTTPS_PROXY http://proxy_host:port
 NO_PROXY localhost,127.0.0.1,.example.com
 ```
 
+**Please note that the `HTTPS_PROXY` is followed by a proxy address which begins with
+`http://` and NOT `https://`.
+Unknowingly setting otherwise is known to cause hard to trace errors.**
```

Comment:
I think we should just tell them to be careful to double-check their work, since http/https errors aren't the only mistake possible here.